parse.js

```diff
@@ -5,11 +5,10 @@ const fs = require('fs');
 const simpleParser = require('mailparser').simpleParser;
 var mailattach = []
 
-// mailattch = [{ filename: '', content: '' }, {  }]
-// regex to get email address inside the brackets <>. 
+// regex to get email address from string 
 // Because mail.email returns email: 'Hello world! <helloworld@gmail.com>'
 
-var regExp = /\<([^)]+)\>/;
+var regExp = /([a-zA-Z0-9._-]+@[a-zA-Z0-9._-]+\.[a-zA-Z0-9_-]+)/;
```

Comment:
email id can have `+`. I don't think this regex supports it. check https://stackoverflow.com/questions/46155/how-to-validate-an-email-address-in-javascript and https://www.regextester.com/19

Reply:
https://regex101.com/r/tF2Dqu/1 
i have edited the current regex to accept + sign.

Reply:
Why are we validating the email? How does it matter what the email was?

Reply:
Ah, I see. Just take whatever is between the brackets, no? Why a regex inside the brackets? Are there cases where the address string contains multiple pieces of text inside angle brackets? Even if that’s the case, can just take the last one until we have contradictory evidence.

Reply:
@sudhirj Not sure the email address is always enclosed with angle/square brackets. So thought of parsing the email address. But now I realise we can craft regex in such way to parse entire string if the angle/square brackets are not there. @sanjeevsiva17